datastore/src/main/java/io/spine/server/storage/datastore/DsRecordStorage.java

```diff
@@ -245,7 +246,10 @@ public EntityRecord apply(@Nullable Entity input) {
         );
         Iterator<EntityRecord> result = emptyIterator();
         for (StructuredQuery<Entity> query : queries) {
-            final Iterator<EntityRecord> records = queryAll(typeUrl, query, fieldMask);
+            final Iterator<EntityRecord> records = queryAll(typeUrl,
```

Comment:
The method, in which body this comment resides, is already way too complex and needs to be refactored into something much more readable. I suggest to have a few well-commented methods instead.